src/System/TimedScopes/TimedScope.cs

```diff
@@ -58,9 +58,9 @@ public class TimedScope : IDisposable
 		/// <param name="correlationData">Correlation data</param>
 		/// <param name="machineInformation">Machine Information</param>
 		/// <returns>Newly created scope</returns>
-		public static TimedScope Create(TimedScopeDefinition scopeDefinition, CorrelationData correlationData, IMachineInformation machineInformation,
+		internal static TimedScope Create(TimedScopeDefinition scopeDefinition, CorrelationData correlationData, IMachineInformation machineInformation,
```

Comment:
>internal [](start = 2, length = 8)

An alternative in many cases is to make the class `internal` and leave this `public`. Would this work here, given the existing use cases? The advantage is that it better hides the implementation if you don't have external dependencies

Reply:
TimedScoped instance need to be returned in public method, so it can't be internal

Reply:
Thanks for confirming